content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -1,37 +1,58 @@
 +++
```

Comment:
Looks like the expandable sections contributed to an unideal table page TOC on the right side. Is there a way to address this?

Reply:
Attempting a workaround.